test/prx_auth/scope_list_test.rb

```diff
@@ -70,8 +70,17 @@ def new_list(val)
       sl = new_list('one two') - nil
       assert sl.contains?(:one) && sl.contains?(:two)
     end
+
+    it 'maintains dashes and capitalization in the result' do
+      sl = new_list('The-Beginning the-middle the-end') - new_list('the-Middle')
+      assert sl.to_s == 'The-Beginning the-end'
```

Comment:
Ah, so is this why tokens now have `read_private` in their `"scope"` instead of `read-private`?  Not that we ever look for those anyways.

Reply:
yeah, that's why (though they are equivalent for the purposes of checking if a scope is present in a token, the string representation is different so pulling a list in tests was slightly more of a pain).